mig/shared/functionality/cat.py

```diff
@@ -46,6 +46,18 @@
 from mig.shared.validstring import valid_user_path
 
 
+def _check_sizes(configuration, paths):
```

Comment:
Minor: could we call this function something a bit more telling and perhaps matching the above variable - like `check_serve_max_bytes` maybe? A docstring explaining that it checks that the accumulated file sizes don't exceed configured variable value would also be nice.

Reply:
renamed plus added docsting - see https://github.com/ucphhpc/migrid-sync/pull/131/files#diff-8f9348586dcddc425f3f8faa0164713569db12d4c9111a3c65d281f3ec973ce8R50